docker/test/upgrade/run.sh

```diff
@@ -77,6 +77,12 @@ remove_keeper_config "async_replication" "1"
 # create_if_not_exists feature flag doesn't exist on some older versions
 remove_keeper_config "create_if_not_exists" "[01]"
 
+# latest_logs_cache_size_threshold setting doesn't exist on some older versions
+remove_keeper_config "latest_logs_cache_size_threshold" "[[:digit:]]\+"
```

Comment:
Not sure that it works - https://s3.amazonaws.com/clickhouse-test-reports/60031/89eb2a38c0a7c113b7b0a96c8069fea6eb48ae9d/bugfix_validation.html

>2024.02.23 06:12:59.294500 [ 757 ] {} <Error> Application: Code: 115. DB::Exception: Unknown setting latest_logs_cache_size_threshold: in Coordination settings config. (UNKNOWN_SETTING), Stack trace (when copying this message, always include the lines below):

Reply:
Because this is for upgrade tests and not for bugfix validation
Need to remove it there too

Reply:
Oh, you are right - https://github.com/ClickHouse/ClickHouse/pull/60375

P.S. too much checks, easy to mess up...